plugins/catalog/src/components/HasComponentsCard/HasComponentsCard.tsx

```diff
@@ -76,7 +76,7 @@ export const HasComponentsCard = ({ variant = 'gridItem' }: Props) => {
         <div>
           No component is part of this system.{' '}
           <Link to="https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component">
-            Learn how to add components.
+            (Learn how to add components.)
```

Comment:
I think it somewhat looks strange with parentheses inside the link. And with a dot inside the parentheses.